src/Tribe/Freemius.php

```diff
@@ -15,8 +15,9 @@ class Tribe__Freemius {
 	private $instances = [];
 
 	/**
-	 * Loading of the vendor files for Freemius vendor
-	 * Freemius class should only be loaded once since it will be registred as a Singleton
+	 * Load the vendor files for Freemius vendor.
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: dfe9945eda892ec8f9f83c999b4db315</sub>